pkg/cmd/pkg/package_describe_test.go

```diff
@@ -0,0 +1,205 @@
+package pkg
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/dcos/client-go/dcos"
+	"github.com/dcos/dcos-core-cli/pkg/cosmos"
+	"github.com/dcos/dcos-core-cli/pkg/cosmos/mocks"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/dcos/dcos-cli/pkg/mock"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_pkgDescribe(t *testing.T) {
```

Comment:
I personally don't like underscores in function names too much. Also IMHO test names should reflect what the test does.

Reply:
Fixed